hal/armv7m/imxrt/10xx/console.c

```diff
@@ -19,9 +19,9 @@
 
 #include "cpu.h"
 #include "console.h"
-#include "imxrt.h"
-#include "../../include/errno.h"
-#include "../../include/arch/imxrt.h"
+#include "imxrt10xx.h"
+#include "../../../../include/errno.h"
+#include "../../../../include/arch/imxrt.h"
```

Comment:
Eh. Maybe we should add -I include directory to kernel CFLAGS. @pawelpisarczyk what do you think?

Reply:
Let's leave it as is.